documentation/frontend/code-style.md

```diff
@@ -36,6 +36,8 @@ The main things to focus on are:
 - don't abuse (or maybe even use) abbreviations
 - think about searchability
 
+Also, `camelCase` should be used by default. Objects coming in from outside sources, such as API response payloads, may use `snake_case`, and there is no need to avoid using snake case in those instances, but any newly defined variables within the NextJS app should use camel case.
```

Comment:
```suggestion
Also, `camelCase` should be used by default. Objects coming in from outside sources, such as API response payloads, may use `snake_case`, and there is no need to avoid using snake case in those instances, but any newly defined variables within the Next.js app should use camel case.
```

extreme nit, "Next.js" is how next describes itself https://nextjs.org/